starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1374,22 +1375,37 @@ sub check_star_catalog {
 	# Bad Pixels ACA-025
         my @close_pixels;
         my @dr;
-	if ($type ne 'ACQ' and $c->{"GS_PASS$i"} =~ /^1|\s+|g[1-2]/) {
+        my @dz;
+        my @dy;
+	if ($type ne 'MON' and $type ne 'FID'){
 	    foreach my $pixel (@bad_pixels) {
 		my $dy = abs($yag-$pixel->{yag});
 		my $dz = abs($zag-$pixel->{zag});
 		my $dr = sqrt($dy**2 + $dz**2);
-		next unless ( $dz < $dither+25 and $dy < $dither+25 );
+		next unless ( $dz < $c->{"HALFW$i"} and $dy < $c->{"HALFW$i"});
```

Comment:
It seems this is reducing the box check size by `$dither` for guide stars.  Should this expression just be adding `$dither`?

Reply:
The intent here was to use the bigger acquisition box to save more bad pixels for checking in the next conditional.   You are absolutely correct that this probably backfires for GUI stars which do have HALFW defined (and it is going to be smaller than dither + 25).  I think the change at this line is not necessary at all if we are getting rid of the checking for bad pixels in acq boxes.